webapp/channels/src/components/admin_console/team_channel_settings/channel/details/channel_profile.tsx

```diff
@@ -32,9 +31,12 @@ export const ChannelProfile = (props: ChannelProfileProps): JSX.Element => {
     if (channel.shared && channel.type) {
         sharedBlock = (
             <div className='channel-organizations'>
-                <FormattedMarkdownMessage
-                    id='admin.channel_settings.channel_detail.channelOrganizations'
-                    defaultMessage='**Organizations**'
+                <FormattedMessage
+                    id='admin.channelSettings.channelDetail.channelOrganizations'
+                    defaultMessage='<b>Organizations</b>'
```

Comment:
Instead of adding the `b` tag to the translated text here, could you actually move the tags outside of the `FormattedMessage`? That way, translators won't have to deal with the formatting tags in this case. We should still use this pattern when part of a sentence needs to be bolded, but that's not the case here

Reply:
@Camillarhi we need to do that for others as well